src/app/src/main/java/com/cmput301f21t09/budgetprojectname/HabitModel.java

```diff
@@ -0,0 +1,39 @@
+package com.cmput301f21t09.budgetprojectname;
+import java.util.Date;
+
+//TODO: Add Frequency
+public class HabitModel {
+    final private String title;
+    final private String ID;
+    final private String reason;
+    final private int streak;
+    final private Date startDate;
+    private Date lastCompleted;
+
+    HabitModel(String title, String reason, Date startDate) {
+        this.title = title;
+        this.reason = reason;
+        this.startDate = startDate;
+        //TODO: Get ID from Firestore
```

Comment:
This model does not directly interface with Firestore, it should be given the ID by the component that creates it